src/components/PeopleInfo/PeopleInfo.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+
+import cn from 'classnames';
+import { Person } from '../../types/Person';
+
+type Props = {
+  person: Person;
+  onSelect?: (
```

Comment:
why do you make this function optional, looks redundant for me
```suggestion
  onSelect: (
```